content/source/docs/enterprise/run/index.html.md

```diff
@@ -82,6 +82,27 @@ If you need to temporarily stop runs from being queued, you can lock the workspa
 
 You can find the lock button in the workspace settings page. Locking a workspace requires write or admin access.
 
+## Installing Terraform Providers
+
+### Providers Distributed by HashiCorp
```

Comment:
I'd suggest having one of the Ecosystem team members check that this phrasing works, unless they already have :)

Reply:
AH, good call.